vm/devices/user_driver/Cargo.toml

```diff
@@ -10,6 +10,7 @@ rust-version.workspace = true
 default = ["vfio"]
 vfio = ["vfio_sys", "sparse_mmap"]
 mmio_simulate_fallback = []
+nvme_keepalive = []
```

Comment:
looks like this feature isn't actually used anywhere

Reply:
Leftover, will remove